sentinel/src/transitions/index.js

```diff
@@ -86,7 +86,7 @@ const processChange = (change, callback) => {
   lineage.fetchHydratedDoc(change.id)
     .then(doc => {
       change.doc = doc;
-      infodoc.get(change)
+      return infodoc.get(change)
```

Comment:
You have to return the promise so any rejections are caught in the `catch` below.